src/kem/kyber/libjade_kyber512_avx2/api.c

```diff
@@ -0,0 +1,19 @@
+#include <oqs/rand.h>
```

Comment:
Files matching `kem/.*/.*/.*` are copied from upstream and are not checked by `tests/test_spdx.sh` (presumably to avoid having to patch individual upstream files). Instead the license information for these files is provided by README.md (this PR updates it to include license information for files matching `src/kem/kyber/libjade_*`) : https://github.com/open-quantum-safe/liboqs?tab=readme-ov-file#license

Reply:
Makes sense! Thanks.